backend/backend.go

```diff
@@ -27,6 +27,7 @@ import (
 type Backend interface {
 	AddrRequest(addr *deriver.Address)
 	AddrResponses() <-chan *AddrResponse
+	TxRequest(txHash string)
 	TxResponses() <-chan *TxResponse
 
 	Finish()
```

Comment:
Finish() should no longer be needed?

Reply:
We could get rid off it, but it feels weird to me that we're not closing resources properly. I think backend should have a proper `Close()` method if someone wanted to use a backend API outside of beancounter.